### PR TITLE
Laduke/190 windows

### PR DESCRIPTION
**Pull request by laduke**

### CODE REVIEW

File: docs/windows-service-unreachable.md [hunk 1]
```diff
@@ -26,3 +26,21 @@ ZeroTier should now be reachable by the UI client when you restart it.
...
+
+is 9993 in the results?
+
+Try to delete the entry
+
+```sh
+netsh interface portproxy delete help
+```
+
+Or try killing Docker or WSL and see if the problem goes away. Let us know what you find out.
```
Comment (rcoder) [inline docs/windows-service-unreachable.md:46]: Maybe change "let us know" to e.g. [let us know](https://discuss.zerotier.com/c/community-support/8)?

**Review by rcoder — COMMENTED** [review]
Definitely a worthwhile addition. One small suggestion to direct people towards a more specific kind of feedback/follow-up.

File: docs/windows-service-unreachable.md (hunk 1 above)
Comment (laduke) [reply docs/windows-service-unreachable.md:46]: great, now we need a "let us know" page

**Review by rcoder — APPROVED** [review]
🚢

**Pull request merged**